backend/routes.js

```diff
@@ -269,17 +269,16 @@ export default class Servers {
                     const existingUsers = getSetting('USERS');
                     const existingUsernames = pluck('username', existingUsers);
                     let status;
```

Comment:
I guess we might as well move these 3 lines into the next section as well since they aren't used outside of that section's scope at all